modules/@apostrophecms/i18n/ui/apos/components/AposI18nLocalize.vue

```diff
@@ -662,9 +664,10 @@ export default {
         this.toLocalizeChoices = this.toLocalizeChoicesStandalone;
       }
     },
-    close() {
+    close(hasBeenSubmitted = false) {
```

Comment:
A bit confusing, don't think it has anything to do with submit draft feature, so I don't know what this is